src/api/tasks.py

```diff
@@ -22,7 +23,7 @@
 from task_queue.nudges.user_event_nudge import prepare_user_events_nudge
 
 
-def generate_csv_and_email(data, download_type, community_name=None, email=None,filename=None):
+def generate_csv_and_email(data, download_type, community_name=None, email=None,filename=None,from_email=None):
```

Comment:
these shoudl all come from MassEnergize support.  They are requested by Admins from the admin portal.